### PR TITLE
[service.clocknotify] 1.0.1

### PR DESCRIPTION
**Pull request by tuoxin**
### Description
- delete all files and restart
- add a new service
- Show current time when watching video and time goes to o'clock
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by tuoxin** [comment]
delete sub-module and restart

**Comment by enen92** [comment]
Please rename PR as " [script.to.be.updated] version"

**Comment by tuoxin** [comment]
@enen92 is there any problem about my plugins?

File: service.clocknotify/addon.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.clocknotify"
+    name="Clock Notify" version="1.0.1" provider-name="tuoxin">
+  <requires>
+    <import addon="xbmc.python" version="2.24.0"/>
```
Comment (Lunatixz) [inline service.clocknotify/addon.xml]: Python version is jarvis, but your PR is for Krypton

Comment (Lunatixz) [reply service.clocknotify/addon.xml]: Transparent Icons aren't allowed, Could you please change it?

File: service.clocknotify/service.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# -*- coding: cp1252 -*-
```
Comment (Lunatixz) [inline service.clocknotify/service.py]: Probably irrelevant (I don't believe Kodis python interpreter parses this info)... Kodi uses UTF-8 not windows encoding.

File: service.clocknotify/service.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            if(bInMinRange and abs(time.time() - NextNotifyTimeStamp) > DeltaSecs and TimeLabelVisible):
+                TimeLabelVisible = False
+                TimeLabel.setVisible(TimeLabelVisible)
+                utils.log("ClockNotify Hidden ")
+            
+            if(TimeLabelVisible):   
+                TimeLabel.setLabel(time.strftime("%H:%M:%S", time.localtime()))
+            
+            utils.log("ClockNotify sleep " + str(sleepTime))
+            xbmc.sleep(sleepTime)
```
Comment (Lunatixz) [inline service.clocknotify/service.py]: Please replace xbmc.sleep with ` if xbmc.waitForAbort(sleepTime): Break`

**Comment by Lunatixz** [comment]
@enen92 LGTM

File: service.clocknotify/addon.xml (hunk 1 above)
Comment (zag2me) [reply service.clocknotify/addon.xml]: @Lunatixz just jumping in here for a question before I submit my own add-on. How do I make it compatible with lower versions if we need to raise PR against Krypton branch? My Add-on works with Isengard(2.20.0 in Add-on XML) so bit confused about how to do it?

**Comment by enen92** [comment]
@zag2me at the moment any new addition must be submitted to the krypton branch which means the python api version in addon.xml has to match the python api version for krypton. That will make the addon only available for krypton and onwards I'm affraid.
People should keep up with newer releases though

File: service.clocknotify/addon.xml (hunk 1 above)
Comment (Lunatixz) [reply service.clocknotify/addon.xml]: You can leave a lower python api, however Kodi users will not have access to it. If you push to the Krypton branch only Krypton users can see and download the plugin.

**Comment by enen92** [comment]
ping @tuoxin

**Comment by Lunatixz** [comment]
@tuoxin can you please review the recommend changes and update your PR, THX.

**Comment by ronie** [comment]
no response... closing

**Pull request closed without merging**